pkg/controller/bootimage/boot_image_controller.go

```
@@ -69,12 +75,28 @@ type Controller struct {
+
+	capiMachineSetLister              dynamiclister.NamespaceLister
+	capiMachineSetListerSynced        cache.InformerSynced
+	capiMachineDeploymentLister       dynamiclister.NamespaceLister
+	capiMachineDeploymentListerSynced cache.InformerSynced
+
+	// Cache-backed lister for the platform-specific CAPI infrastructure template type.
+	// Wired lazily in Run() after the infra cache has synced, so we know which CRD to watch.
+	capiInfraTemplateLister       dynamiclister.NamespaceLister
+	capiInfraTemplateListerSynced cache.InformerSynced
+
 	mapiStats                  MachineResourceStats
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)